src/core_plugins/kibana/server/lib/__tests__/relationships.js

```diff
@@ -0,0 +1,341 @@
+import expect from 'expect.js';
```

Comment:
This file should probably be inside `server/lib/management/saved_objects/__tests__`